content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -16,173 +16,69 @@ This feature requires Kubernetes version >= 1.18.
 
 This task shows how to provision Workload Certificates
 using a custom certificate authority that integrates with the
-[Kubernetes CSR API](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/).
+[Kubernetes CSR API](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/). Different workloads can get their certificates signed from different cert-signers. Each cert-signer is effectively a different CA. It is expected that workloads whose certificates are issued from the same cert-signer can talk MTLS to each other while workloads signed by different signers cannot.
 This feature leverages [Chiron](/blog/2019/dns-cert/), a lightweight component linked with Istiod that signs certificates using the Kubernetes CSR API.
 
-This task is split into two parts. The first part demonstrates how to use the Kubernetes CA itself to sign workload certificates.
-The second part demonstrates how to use a custom CA that integrates with the Kubernetes CSR API to sign your certificates.
+For this example, we use [open-source cert-manager](https://cert-manager.io).
+Cert-manager has added [experimental Support for Kubernetes `CertificateSigningRequests`](https://cert-manager.io/docs/usage/kube-csr/) starting with version 1.4.
 
-## Part 1: Using Kubernetes CA
+## Deploy Custom CA controller in the Kubernetes cluster
 
-{{< warning >}}
-Note that this example should only be used for basic evaluation. The use of the `kubernetes.io/legacy-unknown` signer is NOT recommended in production environments.
-{{< /warning >}}
+1. Deploy cert-manager according to the [installation doc](https://cert-manager.io/docs/installation/).
+   {{< warning >}}
+   Note: Make sure to enable feature gate: `--feature-gates=ExperimentalCertificateSigningRequestControllers=true`
+   {{< /warning >}}
 
-### Deploying Istio with Kubernetes CA
+1. Create three self signed cluster issuers `istio-system`, `foo` and `bar` for cert-manager.
+   Note: Namespace issuers and other types of issuers can also be used.
 
-1. Deploy Istio on the cluster using `istioctl` with the following configuration.
+## Export root certificate for cluster issuer
```

Comment:
Maybe certificate**s** for **each** cluster issuer or for **the** cluster issuer**s**

Reply:
Change to `certificates for each cluster issuer`